drivers/flash/flash_mspi_nor_devs.c

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (c) 2025 Tenstorrent AI ULC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __FLASH_MSPI_NOR_DEVS_H__
+#define __FLASH_MSPI_NOR_DEVS_H__
```

Comment:
This is kind of unusual - is this file being included into another .c file? Since all of the objects are `const` rather than `static const` it could just be compiled.

Reply:
No, this file started as a header file- I'll remove the include guards